### PR TITLE
Migrate CI build from Travis to Github Actions

### PR DESCRIPTION
**Pull request by hbenali**

### CODE REVIEW

**Review by ahamdi — DISMISSED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+jobs:
+  build:
+    name: Build eXo iOS Project
+    runs-on: macos-latest
```
Comment (ahamdi) [inline .github/workflows/build.yml:11]: We may need to put the exact traget version as ios 12

Comment (hbenali) [reply .github/workflows/build.yml]: OK I use Catalina. Please check the compatibility matrix https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md#installed-sdks

Comment (wbenabdessalem) [reply .github/workflows/build.yml]: You can set as minimum macOS 10.15 Catalina

Comment (hbenali) [reply .github/workflows/build.yml]: Yes it is done

Comment (ahamdi) [reply .github/workflows/build.yml]: @hbenali thanks

**Review by ahamdi — APPROVED** [review]

**Pull request merged**